lib/perl/Genome/Qc/Tool/Picard.pm

```diff
@@ -0,0 +1,53 @@
+package Genome::Qc::Tool::Picard;
+
+use strict;
+use warnings;
+use Genome;
+
+class Genome::Qc::Tool::Picard {
+    is => 'Genome::Qc::Tool',
+    is_abstract => 1,
+};
+
+sub cmd_line {
+    my $self = shift;
+    my $cmd = $self->gmt_class->create($self->gmt_params);
+    return $cmd->build_cmdline_list;
+}
+
+sub get_metrics {
+    my $self = shift;
+
+    my $file = $self->output_file;
+    my $gmt_class = $self->gmt_class;
+    my %metrics = $self->metrics;
+    my %metric_results = %{$gmt_class->parse_file_into_metrics_hashref($file)};
```

Comment:
Is this being de-referenced and re-referenced in order to make a copy?  Otherwise it could be passed along.